data/campaigns/Under_the_Burning_Suns/utils/animation_macros.cfg

```diff
@@ -22,7 +22,5 @@ misc/blank-hex.png~BLIT(units/quenoth/fire/flame-color.png,{X},{Y2})~MASK(units/
 
 # Not all unwalkable, but still supposed to provide some visual cue that movement is different.
 # This is only used for animation, it's the list of terrains on which the Sun Sylph will fly.
-# Both the old ^Uf* and new ^Tf* mushrooms are supported here - if the Quenoth elves are moved
-# to core then it seems sensible to still have the old behavior on old maps.
```

Comment:
I think this should stay in, it applies to old UMC maps even when there are no ^Ufs in mainline.

Reply:
Alright, I'll push an update.